lib/web_ui/lib/src/engine/window.dart

```diff
@@ -141,6 +141,9 @@ class EngineWindow extends ui.Window {
             domRenderer.setTitle(arguments['label']);
             domRenderer.setThemeColor(ui.Color(arguments['primaryColor']));
             return;
+          case 'SystemSound.play':
+            // Noop. Fixes https://github.com/flutter/flutter/issues/43462
```

Comment:
No need to reference a fixed issue in the comment. Just say that on the web there are no default system sounds

Reply:
Yep, it'll be associated to the commit # anyway.